python/lib/dependabot/python/python_versions.rb

```diff
@@ -9,14 +9,15 @@ module PythonVersions
 
       # Due to an OpenSSL issue we can only install the following versions in
       # the Dependabot container.
```

Comment:
Here's the commit that added this: https://github.com/dependabot/dependabot-core/commit/e43c80f6e4521fbf59b01c6b97ea8071c359dead

So when this note was added, the vulnerable versions were in the 3.5.x series... 3.6 is now EOL from upstream Python, although we're unofficially made a number of choices to continue to support it until Ubuntu 18.04 goes EOL this coming April.

But at that point, we may want to refactor and drop this note.

Reply:
I filed https://github.com/dependabot/dependabot-core/issues/6044 to track this, as not something I have time to look into right now. For now IMO we should merge this to unblock ourselves/get correct, then later circle back and clean this up.